assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -36,95 +36,73 @@ import { STORE_NAME, ACCOUNT_CREATE, PROPERTY_CREATE, FORM_ACCOUNT_CREATE } from
 import { STORE_NAME as CORE_USER } from '../../../googlesitekit/datastore/user/constants';
 import { STORE_NAME as CORE_FORMS } from '../../../googlesitekit/datastore/forms/constants';
 import { actions as tagActions } from './tags';
-const { createRegistrySelector, createRegistryControl } = Data;
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+const { createRegistrySelector } = Data;
+
+const fetchGetAccountsPropertiesProfilesStore = createFetchStore( {
+	baseName: 'getAccountsPropertiesProfiles',
+	controlCallback: ( { data } ) => {
+		return API.get( 'modules', 'analytics', 'accounts-properties-profiles', data, {
+			useCache: false,
+		} );
+	},
+	reducerCallback: ( state ) => {
+		// Actual accounts, properties, profiles are set by resolver with
+		// custom logic, hence here we just set a flag.
+		return {
+			...state,
+			isAwaitingAccountsPropertiesProfilesCompletion: true,
```

Comment:
I'm not sure this flag is necessary. See my comments in the properties/profile stores, but I think we can get the same data with built-in resolution selectors which is better since it's args specific (that doesn't matter as much for this one since there are no args, but I don't think it's needed.